protocol/migrations/sqlite/1674210659_add_contact_request_local_clock.up.sql

```diff
@@ -0,0 +1,16 @@
+
+ALTER TABLE contacts ADD COLUMN contact_request_local_clock INT;
```

Comment:
Not sure, operationally is a bit more complex since you have to deal with JOINs etc, so I tend to prefer to add to the `contacts` table, but I am not really an expert on sql database design, I don't have a strong opinion on the matter myself.

Reply:
Cool, no strong preference myself, just leaning a bit more on normalization. It's helpful for avoiding duplicate data across tables for example, but I don't even know if this is a problem in `status-go`, probably not. There are other benefits that could outweigh the increased complexity in queries. I'm no expert either. Thanks Andrea